libc/newhdrgen/yaml/search.yaml

```diff
@@ -57,3 +57,13 @@ functions:
     return_type: void
     arguments:
       - type: void *
+  - name: lfind
+    standards:
+      - POSIX
+    return_type: void *
+    arguments:
+      - type: void *
+      - type: void *
```

Comment:
These first two parameters need to be `const void *`.

If you do a full build, you can `find <build dir> -name search.h` and check that the function signature generated matches the spec.

Reply:
Resolved by [5b4b7ca](https://github.com/llvm/llvm-project/pull/114692/commits/5b4b7ca9afbcaf1e216cd605a54fd41f1fb4108e)